Dockerfile-core.template

```diff
@@ -0,0 +1,42 @@
+FROM bids/base_validator
```

Comment:
This is the only BIDS-related item: why not base directly on,e.g., ubuntu?

Reply:
I have no idea why this is the base image. Is it containing something that BIDS-Apps expect?

Reply:
Can't remember now but  given the `--skip-bids-validator` flag, I assume that a BIDS-App would by default validate the BIDS input and therefore expect `bids-validator` to be available.

Reply:
`bids/base_validator` *is* ubuntu with bids-validator instally. But it seems to be very outdated as well: https://github.com/BIDS-Apps/base_validator/blob/master/Dockerfile

Reply:
That one could probably be based on `node:current-slim`

Reply:
If `bids/base_validator` is meant to be a base image on which to base BIDS app then `node:current-slim` would probably not be first choice. Even [fmriprep](https://github.com/poldracklab/fmriprep/blob/master/Dockerfile) is not based on `bids/base_validator` but a very specific image (`ubuntu:xenial-20200114`). Not sure one can impose an base, therefore `bids/base_validator` and `bids/matlab-compiler-runtime` are more examples than base images?

Reply:
> Not sure one can impose an base, therefore bids/base_validator and bids/matlab-compiler-runtime are more examples than base images?

perhaps, I'll ask about this in an upcoming meeting with other BIDS folks who may know.